front/src/common/redux/root-saga.js

```diff
@@ -1,6 +1,6 @@
 import { fork } from 'redux-saga/effects';
-import { rootAuthSaga } from './auth/sagas/auth.saga';
+import { rootSaga } from './auth/auth.saga';
```

Comment:
It is better to name it as `authSaga` because it is much more explicit and won't create conflicts in the future with other sagas

Reply:
done